daemon/api_test.go

```diff
@@ -2788,6 +2788,8 @@ func (s *apiSuite) TestInterfaces(c *check.C) {
 
 func (s *apiSuite) TestConnectPlugSuccess(c *check.C) {
 	d := s.daemon(c)
+	command := testutil.MockCommand(c, "snap", "")
```

Comment:
Why are we mocking snap? Is it because of `snap run`? Are there any hooks being called here?